tests/models/data/horovod/train_default_model.py

```diff
@@ -0,0 +1,36 @@
+import argparse
+import json
+import os
+import sys
+
+import horovod.torch as hvd
+
+PATH_HERE = os.path.abspath(os.path.dirname(__file__))
+PATH_ROOT = os.path.join(PATH_HERE, '..', '..', '..', '..')
+sys.path.insert(0, os.path.abspath(PATH_ROOT))
+
+from pytorch_lightning.callbacks import ModelCheckpoint  # noqa: E402
+import tests.base.utils as tutils  # noqa: E402
+
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--trainer-options', required=True)
+
+
+def test(trainer_options):
```

Comment:
test what?

Reply:
Renamed for clarity and added a docstring at the top of the file to explain usage.